src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -889,6 +901,32 @@ class PublishArgument(cli.Argument):
             'required': False,
         },
     )
+    EXPIRE_BUILDS = cli.ArgumentProperties(
+        key='expire_builds',
+        flags=('--expire-builds',),
+        type=bool,
+        description=(
+            'Expires all builds other than the current for the application '
+            'before submitting the build to TestFlight.'
+        ),
+        argparse_kwargs={
+            'required': False,
+            'action': 'store_true',
+        },
+    )
+    CANCEL_PREVIOUS_SUBMISSIONS = cli.ArgumentProperties(
```

Comment:
Similarly to `EXPIRE_BUILDS`, this is declared not only for `app-store-connect publish`, but also for `app-store-connect builds submit-to-app-store`, which doesn't include it in signature.